proto/umee/leverage/v1/query.proto

```diff
@@ -356,3 +361,30 @@ message QueryLiquidationTargetsRequest {}
 message QueryLiquidationTargetsResponse {
   repeated string targets = 1;
 }
+
+// QueryMarketSummaryRequest defines the request structure for the
+// MarketSummary gRPC service handler.
+message QueryMarketSummaryRequest {
+  string denom = 1;
+}
+
+// QueryMarketSummaryResponse defines the response structure for the
+// MarketSummary gRPC service handler.
+message QueryMarketSummaryResponse {
+  string symbol_denom = 1;
+  uint32 exponent = 2;
+  string oracle_price = 3
+  [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec", (gogoproto.nullable) = true];
+  string uToken_exchange_rate = 4
+  [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec", (gogoproto.nullable) = false];
+  string lend_APY = 5
+  [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec", (gogoproto.nullable) = false];
+  string borrow_APY = 6
+  [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec", (gogoproto.nullable) = false];
+  string market_size = 7
+  [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int", (gogoproto.nullable) = false];
```

Comment:
Note `market_size` (tokens) not `market_size_usd` here. See discussion question in PR description